airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2280,6 +2304,25 @@ paths:
                 $ref: "#/components/schemas/ImportRead"
         "404":
           $ref: "#/components/responses/NotFoundResponse"
+  /v1/internal/set_temporal_workflow_in_attempt:
```

Comment:
@cgardens discussion from parent PR:
![image](https://user-images.githubusercontent.com/7798112/186285203-f340ba87-f479-4ef8-b79b-cd81f66d4fc5.png)

Reply:
I agree that this is a good idea to rename, @xiaohansong I'll let you address the various suggestions made around the internal API code

Reply:
Yep working on that

Reply:
Xiaohan fixed this up!